database/src/library.rs

```diff
@@ -184,4 +184,23 @@ impl InsertableLibrary {
 
         Ok(lib_id)
     }
+
+    pub async fn check_duplicates(
```

Comment:
I think this should be a method for `Library`, we should also rename it to `is_scanned`. Then we can directly pass to it a vector of strings.

Reply:
On a Library, we will need a different implementation, which executes:
"SELECT count(*) as cnt FROM indexed_paths WHERE location = $1 and library_id <> $2" - to actually find paths in different libraries.